src/main/java/com/example/springbootrestexample/configurations/AppConfig.java

```diff
@@ -0,0 +1,46 @@
+package com.example.springbootrestexample.configurations;
+
+import com.example.springbootrestexample.repositories.*;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AppConfig {
+
+    @Bean(name="bookRepository")
```

Comment:
It is not a good practice to put names to the beans. It is enough information with the type (interface)